src/lib/components/index/hero.svelte

```diff
@@ -1,28 +1,50 @@
+<script>
+  import PrimaryButton from "../library/primary-button.svelte";
+</script>
+
 <div
-  class="mt-x-large relative w-11/12 md:w-fit mx-auto justify-between text-left md:text-left md:mb-[400px]"
+  class="mt-x-large relative w-11/12 md:w-fit mx-auto justify-between text-left md:mb-[420px]"
 >
-  <h1 class="text-5xl md:text-6xl font-extrabold md:ml-36">
-    CDE <span class="font-light">universe</span>
-  </h1>
-  <div class="mt-4 text-xl md:text-[28px] md:leading-[2.5rem] md:ml-36">
-    June 1 and 2
-  </div>
-  <div
-    class="mt-1 font-medium text-rose text-xl md:text-[28px] md:leading-[2.5rem] md:ml-36"
-  >
-    <div>In-Person, Pebblebed</div>
-    <div>San Francisco, CA</div>
+  <div class="md:ml-36">
+    <h1 class="text-5xl md:text-6xl font-extrabold">
+      CDE <span class="font-light">universe</span>
+    </h1>
+
+    <div class="mt-4 text-xl md:text-[28px] md:leading-[2.5rem]">
+      June 1 and 2
+    </div>
+
+    <div
+      class="mt-1 font-medium text-rose text-xl md:text-[28px] md:leading-[2.5rem]"
+    >
+      <div>Pebblebed</div>
+      <div>San Francisco, CA</div>
+    </div>
+
+    <div class="mt-6">
+      <PrimaryButton
+        href="https://www.eventbrite.co.uk/e/cde-universe-2023-cloud-development-conference-by-gitpod-tickets-626052498877"
+        text="Buy ticket"
```

Comment:
Sometimes we use "ticket**s**" and other times "ticket". Is that on purpose? I'd suggest that we globally align the CTA 
@MichaelAring